### PR TITLE
Added 'Run Highlight Models' Service to Pipeline

### PR DESCRIPTION
**Pull request by ewega**
Self-Explanatory. 

I'll add the post processing to the pipeline once this one is approved.
Lmk if i fucced up

### CODE REVIEW

File: services/runModels.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import numpy as np
+import pandas as pd
+import pickle
+
+def getRandomForestPredictions(X):
+  rf_model = pickle.load(open('highlight-model/rf_100t.sav', 'rb')) # Get model from file
```
Comment (tjlam) [inline services/runModels.py]: nit: maybe we should put the model files in a the datastore folder for consistency?

File: run.py [hunk 2]
```diff
@@ -73,4 +74,15 @@
...
+
+
+# Run our highlight models
+print("Running our Models")
+features_df = df[['pitch', 'amplitude', 'subjectivity', 'polarity', 'pred_excitement']]
+print("Running Random Forest..")
+rf_predictions = runModels.getRandomForestPredictions(features_df)
+print("Running Neural Network..")
+nn_predictions = runModels.getNeuralNetworkPredictions(features_df)
+
```
Comment (tjlam) [inline run.py:89]: add these columns to the big boi dataframe

File: services/runModels.py [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+  output = rf_model.predict(X)
+  
+  return np.array(output)
+
+def getNeuralNetworkPredictions(X):
+  nn_model = pickle.load(open('nn_2layers_0.5.sav', 'rb')) # Get model from file
+  output = nn_model.predict_classes(X)
+  
+  return np.array(output)
```
Comment (tjlam) [inline services/runModels.py]: new line

**Review by tjlam — CHANGES_REQUESTED** [review]
Just some smol changes

File: run.py (hunk 2 above)
Comment (ewega) [reply run.py:89]: Fixed

File: services/runModels.py (hunk 1 above)
Comment (ewega) [reply services/runModels.py]: Fixed

File: services/runModels.py (hunk 3 above)
Comment (ewega) [reply services/runModels.py]: Fixed

**Review by tjlam — APPROVED** [review]

**Review by mattlianje — APPROVED** [review]
Solid like the walls of Ba-Sing-Sei)

**Pull request merged**